pygmt/x2sys.py

```diff
@@ -136,25 +137,25 @@ def x2sys_init(tag, **kwargs):
         - **c** - Cartesian userdist or userdist/usertime
```

Comment:
Lines 130-131 are very confusing. I suggest follow the GMT documentation, i.e, **d**|**s***unit*. And *unit* can be **e**, **f** et al.

Reply:
Good point, thanks.